src/app/components/Calendar/CustomCalendar.css

```diff
@@ -0,0 +1,126 @@
+.react-calendar {
+  width: 320px;
+  max-width: 100%;
+  background: white;
+  border: none;
+}
+
+/* Adapt Nav Bar // Is there a way to overwrite the inline style of the navigation that sets display=flex to grid, other that using !important? */
+.react-calendar__navigation {
+  display: grid !important;
```

Comment:
Does it not work without `!important`? Otherwise, maybe see what element it is and make the selector more specific